src/common/security/utils/ram_role.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"github.com/goharbor/harbor/src/common"
+	"github.com/goharbor/harbor/src/common/ram"
+)
+
+var (
+	rolePoliciesMap = map[string][]*ram.Policy{
+		"projectAdmin": {
+			parsePolicy("image, all"), // compatible with security all perm of project
```

Comment:
I don't think we need action "all"  for one particular resource?  
If we need all perm it should be `*, *`?

Reply:
It is only needed when compatible to implement current security `HasAllPerm`, when `HasReadPerm`, `HasWritePerm` and `HasAllPerm` methods are removed from `security.Context` interface, `image, all` policy will be removed from projectAdmin policies.

Reply:
resources  and actions for project now using const, current only provide image resource, pull, push, pushpull etc vars